src/sentry/api/bases/organization.py

```diff
@@ -516,7 +517,11 @@ def get_filter_params(
         return params
 
     def convert_args(
-        self, request: Request, organization_slug: str | None = None, *args: Any, **kwargs: Any
+        self,
+        request: Request,
+        organization_slug: str | int | None = None,
```

Comment:
can an organization slug be an `int`? That doesn't seem right, we would want an error raised if the types are wrong

Reply:
oh because this can be the org id instead? I think we should add some doc strings and comments here explaining why and what this is

Reply:
I'll add the docstring mentioning the temporary change